src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars__container">
+        <div class="stars stars--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="stars stars--1">
```

Comment:
add one empty line between same level elements
```suggestion
        </div>
        
        <div class="stars stars--1">
```